tools/buildPackage.js

```diff
@@ -19,11 +19,24 @@ var env = {
   NODE_ENV: 'production'
 }
 
+console.log('Cleaning build target')
+
+if (isWindows) {
+  var cmds = [
+    'if exist ' + buildDir + ' rmdir /s /q ' + buildDir,
+  ]
+  execute(cmds, env, console.log.bind(null, 'done'))
+} else {
+  var cmds = [
+    'rm -rf ' + buildDir,
+    'rm -f dist/*.dmg dist/*.nupkg dist/*.exe dist/*.msi dist/RELEASES dist/*.zip',
+  ]
+  execute(cmds, env, console.log.bind(null, 'done'))
+}
```

Comment:
Also no need to have a console message when the clean is done, it would be a bit misleading that way.

The easiest way to fix is to mutate the cmds array.